### PR TITLE
update workflow branch

### PR DESCRIPTION
**Pull request by acedeywin**
## Description

- Update Workflow

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added

### Related Issues
Fixes CHI-1259

### CODE REVIEW

File: .github/workflows/deploy-helplines-parallel-worflows.yml [hunk 1]
```diff
@@ -14,121 +14,121 @@ on:
...
+    uses: techmatters/serverless/.github/workflows/uk_staging.yml@master
     secrets: inherit
     with:
       changelog: ${{ inputs.changelog }}
      
       
 
   deploy-ukraine_staging: 
-    uses: techmatters/serverless/.github/workflows/ukraine_staging.yml@CHI-1259_workflow_to_deploy_serverless_helpline_stg_norerun
+    uses: techmatters/serverless/.github/workflows/ukraine_staging.yml@master
```
Comment (stephenhand) [inline .github/workflows/deploy-helplines-parallel-worflows.yml:117]: Can you update this to `hungary_staging.yml` as well?

**Review by stephenhand — COMMENTED** [review]
Do we have to set the branch explicitly branch to @master? Can we not leave it as the default?

If we deploy from a tag (which will be typical going forward) and the workflow on master has changed, this could unexpected behaviour

@GPaoloni - do we have to specify a branch here if the workflow is from the same repo as the parent?

File: .github/workflows/deploy-helplines-parallel-worflows.yml (hunk 1 above)
Comment (acedeywin) [reply .github/workflows/deploy-helplines-parallel-worflows.yml]: @stephenhand It's done.

**Review by stephenhand — APPROVED** [review]
LGTM

**Pull request merged**